IMDb/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "manish",
+        "id": "332177981214818315"
+    },
+    "service": "IMDb",
+    "description": {
+        "en": "IMDb (Internet Movie Database) is an online database of information related to films, television programs, home videos, video games, and streaming content online – including cast, production crew and personal biographies, plot summaries, trivia, fan and critical reviews, and ratings."
+        },
+    "url":"www.imdb.com",
+    "version": "1.0",
+    "logo": "https://i.imgur.com/OCAWBbv.png",
+    "thumbnail": "https://i.imgur.com/OCAWBbv.png",
```

Comment:
Thumbnail should be a preview of the website

Reply:
Made changes!